onnxruntime/core/providers/partitioning_utils.cc

```diff
@@ -299,6 +299,10 @@ std::unique_ptr<ComputeCapability> MakeComputeCapability(const GraphViewer& grap
   meta_def->status = ONNX_NAMESPACE::EXPERIMENTAL;
 
   for (const auto& input : ordered_subgraph_inputs) {
+    if (graph_viewer.IsConstantInitializer(input->Name(), true)) {
+      meta_def->constant_initializers.push_back(input->Name());
+    }
+
     meta_def->inputs.push_back(input->Name());
```

Comment:
This should be an `else` for adding to inputs. By definition a constant initializer is an initializer that does NOT have a matching graph input. By adding it to inputs as well, you're making it a non-constant initializer, as an input would override the value in the initializer.

Reply:
It "Should" be. But There would be too many references of `inputs` in the repo, and the "inputs" has different semantic meanings.